### PR TITLE
ToDo-List: Peer feedback

### PR DESCRIPTION
**Pull request by kessie2862**
In this branch, I

- [ ] Added linter and webpack config files
- [ ] Provided basic styles for the form
- [ ] Added functionalities for adding, removing, editing, and clearing completed tasks
- [ ] Used webpack to bundle Javascript file

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+function addNewTask(description) {
+  const taskIndex = tasks.length;
+
+  const task = { description, completed: false, index: taskIndex };
+  tasks.push(task);
+  saveTasks();
+
+  const listItemElement = createTaskLists(task);
+  taskList.appendChild(listItemElement);
+}
```
Comment (AbdelazizSoliman) [inline src/index.js]: Hi @kessie2862 

Good job so far!
I suggest to use arrow function, your code will me more clean and followed JavaScript best practice

**Review by AbdelazizSoliman — CHANGES_REQUESTED** [review]
- [ ] No linter errors.
- [ ] Your code is follow DRY, YAGNI, KISS.
- [ ] Your code follow JavaScript best practicing.
Changes required:
Kindly Check your code and make the required changes.

File: src/index.js (hunk 1 above)
Comment (kessie2862) [reply src/index.js]: Thanks. I have fixed it.

**Review by AbdelazizSoliman — APPROVED** [review]
Status Approved
Hi @kessie2862 
You made a great project,
 you project is approved, there is nothing else to say other than you can merge it.
 Congratulations!

**Pull request merged**